pkg/domain/stateDiffGlobalConfig.go

```diff
@@ -0,0 +1,42 @@
+package domain
+
+import (
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/ecosystem"
+)
+
+func determineGlobalConfigDiffs(
+	config GlobalConfig,
+	actualDoguConfig map[common.GlobalConfigKey]ecosystem.GlobalConfigEntry,
+) GlobalConfigDiff {
+	var configDiffs []GlobalConfigEntryDiff
+
+	// present entries
+	for key, expectedValue := range config.Present {
+		actualEntry, actualExists := actualDoguConfig[key]
+		configDiffs = append(configDiffs, determineGlobalConfigDiff(key, string(actualEntry.Value), actualExists, string(expectedValue), true))
+	}
+	// absent entries
+	for _, key := range config.Absent {
+		actualEntry, actualExists := actualDoguConfig[key]
+		configDiffs = append(configDiffs, determineGlobalConfigDiff(key, string(actualEntry.Value), actualExists, string(actualEntry.Value), false))
+	}
+	return configDiffs
+}
+
+func determineGlobalConfigDiff(key common.GlobalConfigKey, actualValue string, actualExists bool, expectedValue string, expectedExists bool) GlobalConfigEntryDiff {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `determineGlobalConfigDiff` is unused